domain-cc/svc-cc-app/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+""" Pytest configuration. This file is automatically loaded by pytest before any tests. """
+
```

Comment:
have no idea what's up w/ `isort` in CI here
Ran isort over my code and it exists w/ no output to console.

<img width="591" alt="Screenshot 2023-05-26 at 10 59 08 AM" src="https://github.com/department-of-veterans-affairs/abd-vro/assets/17188013/d869e1df-dbbc-4212-ad87-06526c0c82b8">

Reply:
weird, good find! Thank you!